### DISCUSSION TITLE
Chakra UI toast https://chakra-ui.com/docs/components/toast is missing in Reflex

### DISCUSSION THREAD

**Discussion by kumaraguru1735** [reflex-dev/reflex#2483] [General]
Chakra UI toast https://chakra-ui.com/docs/components/toast is missing in Reflex

**Comment by picklelo** [comment]
Thanks for raising. Toast is a bit different of a component since it's not a true React component, which is why we didn't wrap it originally. If anyone wants to try wrapping, we will accept a PR for it.
